react-carousel/src/plugins/rtl.js

```diff
@@ -0,0 +1,94 @@
+import { useCallback, useEffect } from 'react';
+import { useRecoilState, useRecoilValue } from 'recoil';
+
+import { pluginNames } from '../constants/plugins';
+import CAROUSEL_STRATEGIES from '../constants/carouselStrategies';
+import clamp from '../tools/clamp';
+import { slidesState } from '../state/atoms/slideAtoms';
+import {
+  carouselValueState,
+  slideMovementState,
+  trackStylesState,
+} from '../state/atoms/carouselAtoms';
+
+import './rtl.scss';
+
+const rtl = ({ carouselProps }) => ({
+  name: pluginNames.RTL,
+  plugin: () => {
+    const [trackStyles, setTrackStyles] = useRecoilState(trackStylesState);
+    const slides = useRecoilValue(slidesState);
+
+    useEffect(() => {
+      if (carouselProps?.children?.length !== slides.length) {
+        setTrackStyles({
+          ...trackStyles,
+          transform: -carouselProps.transformOffset,
+        });
+      }
+    }, [carouselProps.transformOffset]);
+  },
+  strategies: () => {
+    const slides = useRecoilValue(slidesState);
+    const slideMovement = useRecoilValue(slideMovementState);
+    const value = useRecoilValue(carouselValueState);
+
+    return {
+      [CAROUSEL_STRATEGIES.CHANGE_SLIDE]: (originalValue, prevValue) => {
```

Comment:
could you rename `prevValue` into `previousValue` here and in other places (and other files)?

Reply:
I think that everyone understands prev is abbrev from previous